microsoft-edge/devtools-guide-chromium/accessibility/test-inspect-states.md

```diff
@@ -0,0 +1,118 @@
+
+
+## Checking text color contrast in the default state
+
+<!-- Inspect tool: information overlay: Accessibility section: Contrast row -->
+
+In addition to the automatic color-contrast tests in the **Issues** tool, you can also use the **Inspect** tool to check whether individual page elements have enough contrast.  If contrast information is available, the **Inspect** overlay shows the contrast ratio and a checkbox item.  A green check mark icon indicates there's enough contrast, and a yellow alert icon indicates not enough contrast.
+
+For example, the links in the sidebar navigation menu have enough contrast, but the green **Dogs** list item in the **Donation status** section doesn't.
+
+:::image type="complex" source="../media/a11y-testing-enough-contrast.msft.png" alt-text="The links in the sidebar navigation menu have enough contrast, as shown in the Inspect overlay" lightbox="../media/a11y-testing-enough-contrast.msft.png":::
+    The links in the sidebar navigation menu have enough contrast, as shown in the **Inspect** overlay
+:::image-end:::
+
+An element that doesn't have enough contrast is flagged by a warning in the **Inspect** overlay:
+
+:::image type="complex" source="../media/a11y-testing-not-enough-contrast.msft.png" alt-text="An element that doesn't have enough contrast is flagged by a warning in the Inspect overlay" lightbox="../media/a11y-testing-not-enough-contrast.msft.png":::
+    An element that doesn't have enough contrast is flagged by a warning in the **Inspect** overlay
+:::image-end:::
+
+
+## Hovering when the Inspect tool is active doesn't show the text-color contrast for the hover state
+
+The **Inspect** tool's information overlay only represents a single state.  But the elements on the page could have different states, and you need to test those states for accessibility, too.  For example, when you hover the mouse pointer over the menu of the accessibility-testing demo page, you get a rather involved animation that changes the colors:
+
+1.  Open the [accessibility-testing demo webpage][DevToolsA11yErrorsDemopage] in a new tab.
+
+1.  Hover over the blue links in the sidebar navigation menu.  The menu animations run.
+
+    :::image type="complex" source="../media/a11y-testing-hover.msft.png" alt-text="The menu item showing different colors when the mouse pointer is over it" lightbox="../media/a11y-testing-hover.msft.png":::
+        The menu item showing different colors when the mouse pointer is over it
+    :::image-end:::
+    
+    But when you turn on the Inspect tool, you can't reach the `hover` state of the menu element to test the contrast ratio, because the `hover` state in your styles doesn't get triggered when the Inspect tool is active.  When the Inspect tool is active, animations of the demo menu items don't occur when you hover over them.  Confirm this lack of animation as follows:
```

Comment:
```suggestion
Now, use the Inspect tool. When the Inspect tool is used, animations on the menu items won't run when you hover over them.  When using the Inspect tool, you can't reach the `hover` state on menu items to test the contrast ratio, because the `hover` state in your styles isn't triggered.  
    
To confirm that your animations don't run, perform the following steps.
```